### PR TITLE
Fix ksort type

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # 30979

### Summary of Changes
changed to integer per https://www.w3schools.com/php/func_array_ksort.asp

### Testing Instructions
go to global configuration
set debug to yes
set log to development
go to administrator/index.php?option=com_templates&view=template&id=506
observer php warning exists
apply pr
refresh page verify that error is gone.

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request
image

### Documentation Changes Required
none

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -1,37 +0,0 @@
-# Security Policies and Procedures
```
Comment (Llewellynvdm) [inline .github/SECURITY.md:1]: Why is this removed? sorry silly question... but is this not just a fix for ksort?

**Comment by SharkyKZ** [comment]
Unrelated changes aside, this PR is wrong.

File: .github/SECURITY.md (hunk 1 above)
Comment (N6REJ) [reply .github/SECURITY.md:1]: shouldn't even exist. only one file was changed *sigh*
I'll redo this yet again.. I hate when git does this to me.

**Comment by N6REJ** [comment]
@SharkyKZ the fix is NOT wrong.  dunno why you say it is but it's not.

**Comment by N6REJ** [comment]
replaced pr with one thats not corrupted. https://github.com/joomla/joomla-cms/pull/31011

**Comment by SharkyKZ** [comment]
You can't just replace an undefined constant with a random value. Passing an unrecognized flag, my guess, will result in default value being used. `ksort()` simply doesn't support natural sorting on PHP 5.3, no matter what you pass. Not to mention values of constants can change between PHP versions.

**Comment by N6REJ** [comment]
it's not a random value.. read the docs
